server/emails/ExportEmail.js

```diff
@@ -8,25 +8,35 @@ import Footer from "./components/Footer";
 import Header from "./components/Header";
 import Heading from "./components/Heading";
 
-export const exportEmailText = `
+export const exportEmailSuccessText = `
 Your Data Export
 
-Your requested data export is attached as a zip file to this email.
+Your requested data export is completed, please visit the export section to find the link to zip file.
 `;
 
-export const ExportEmail = () => {
+export const exportEmailErrorText = `
```

Comment:
Lets split this into two templates,`ExportSuccessEmail` and `ExportFailureEmail` these are pretty different